From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: George Kottackal <kottackal.george@gmail.com>
Date: Wed, 25 Jan 2023 12:03:54 +0000
Subject: OS-14339: Browser constructor with negative Rect. dimensions.

Browser constructor with custom rectangle dimension,
partly or totally, on screen or off screen. This patch enables
the wayland subsurface based delegated compositing.
This is needed to allow windows to be drawn beyond the
visible screen.

diff --git a/ui/ozone/common/features.cc b/ui/ozone/common/features.cc
index ec153a46080650a55a0aec5f2e20c01539b2d9ab..fd5bdbe7c6288cf58501047e37382cd4abec8cf8 100644
--- a/ui/ozone/common/features.cc
+++ b/ui/ozone/common/features.cc
@@ -17,7 +17,7 @@ BASE_FEATURE(kWaylandOverlayDelegation,
 #if BUILDFLAG(IS_CHROMEOS_LACROS)
              base::FEATURE_ENABLED_BY_DEFAULT
 #else
-             base::FEATURE_DISABLED_BY_DEFAULT
+             base::FEATURE_ENABLED_BY_DEFAULT
 #endif
 );
 
